### PR TITLE
Add TravisCI

### PR DESCRIPTION
**Pull request by abookyun**
* Fix Gemfile error
* Add TravisCI badge
* Just need to implement one last test and it will be done!

Thanks

### CODE REVIEW

**Comment by huacnlee** [comment]
```
.......rm: cannot remove `/home/travis/build/abookyun/rucaptcha/spec/../tmp/*.png': No such file or directory
F
Failures:
  1) OCR should not read by OCR lib
     Failure/Error: File.open(fname, 'w+') do |f|
     Errno::ENOENT:
       No such file or directory @ rb_sysopen - /home/travis/build/abookyun/rucaptcha/spec/../tmp/b7be.png
     # ./spec/ocr_spec.rb:13:in `initialize'
     # ./spec/ocr_spec.rb:13:in `open'
     # ./spec/ocr_spec.rb:13:in `block (3 levels) in <top (required)>'
     # ./spec/ocr_spec.rb:10:in `each'
     # ./spec/ocr_spec.rb:10:in `block (2 levels) in <top (required)>'
```

https://travis-ci.org/abookyun/rucaptcha

**Comment by abookyun** [comment]
I think it's issue for OCR spec not infected to travis itself?

**Comment by huacnlee** [comment]
Yes! That is reason of why I am not added CI.

But you can continue try it.

**Comment by abookyun** [comment]
lol alright then

**Comment by abookyun** [comment]
@huacnlee tried but can't make it right :weary: could we just merge travis then figure out what makes Ci fail?

File: README.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 # RuCaptcha
 
 [![Gem Version](https://badge.fury.io/rb/rucaptcha.svg)](https://badge.fury.io/rb/rucaptcha)
+[![Build Status](https://travis-ci.org/abookyun/rucaptcha.svg)](https://travis-ci.org/abookyun/rucaptcha)
```
Comment (huacnlee) [inline README.md]: Update right URL first

**Comment by abookyun** [comment]
Fixed :)

**Comment by huacnlee** [comment]
Merge multiple commits into one commit.

```
git reset ...
git add -A
git commit 
git push you branch -f
```

**Comment by abookyun** [comment]
Squash them into two commits.

**Comment by abookyun** [comment]
Still have typo :weary: fixed then

**Comment by abookyun** [comment]
Thanks! and sorry for mess commits :sob:

**Pull request merged**